collectd/files/collectd_python.conf

```diff
@@ -7,10 +8,18 @@
   LogTraces false
   Interactive false
 
-  {%- for plugin_name, plugin in plugin.iteritems() %}
-  {%- if (plugin.get('execution', 'local') == 'local' or client.remote_collector) and plugin.get('plugin', 'native') == 'python' %}
+  {%- for plugin_name, plugin in local_plugin.iteritems() %}
+  {%- if plugin.get('plugin', 'native') == 'python' %}
   {%- include plugin.template %}
```

Comment:
s/{%-/{%/ otherwise it generates an invalid configuration

`Interactive falseImport "rabbitmq_info"`